src/applications/CMakeLists.txt

```diff
@@ -25,5 +25,8 @@ set(STARSH_SRC
     "${CMAKE_CURRENT_SOURCE_DIR}/minimal.c"
     "${CMAKE_CURRENT_SOURCE_DIR}/particles.c"
     "${CMAKE_CURRENT_SOURCE_DIR}/cauchy.c"
+    "${CMAKE_CURRENT_SOURCE_DIR}/mesh_deformation/cube.c"
```

Comment:
not sure if there is a naming convention in starsh? @Muxas

Reply:
for me it sounds OK